corehq/apps/app_manager/models.py

```diff
@@ -3987,6 +3988,8 @@ class ApplicationBase(VersionedDoc, SnapshotMixin,
     # always false for RemoteApp
     case_sharing = BooleanProperty(default=False)
 
+    language_profiles = DictProperty()
```

Comment:
The dict actually does use UUIDs as the key (these are generated on creation of a new profile in the `LanguageProfilesView`, but i cant figure out how to link to the line in the PR), and values are another dict, with keys that include `langs`, `name` and whatever else we want to add later. The `name` property is only used in the ui. Happy to fomalize that with a class (or named tuple, etc.) though, if that is better